packages/@aws-cdk/aws-dynamodb-global/lib/aws-dynamodb-global.ts

```diff
@@ -0,0 +1,64 @@
+import dynamodb = require("@aws-cdk/aws-dynamodb");
+import cdk = require("@aws-cdk/cdk");
+import { LambdaGlobalDynamoDBMaker } from "./lambda-global-dynamodb";
+
+/**
+ * Properties for the mutliple DynamoDB tables to mash together into a
+ * global table
+ */
+export interface RegionalTablesProps extends cdk.StackProps, dynamodb.TableOptions {
```

Comment:
Why is this called "RegionalTables" instead of "GlobalTable"?

Reply:
You asked for the class to be renamed?
https://github.com/awslabs/aws-cdk/pull/2082#discussion_r269786560

Reply:
Not the class, the property name. It's: `GlobalTable.regionalTables`. Otherwise it doesn't make sense...